src/path.cr

```diff
@@ -568,9 +569,9 @@ struct Path
     name = @name
```

Comment:
```suggestion
     name = @name
     home ||= Path.home
```

Reply:
Wouldn't that force `Path.home` to always be evaluated? which is precisely what we want to avoid here?

Reply:
Ouch, you're right 👍